src/plenoptic/simulate/canonical_computations/non_linearities.py

```diff
@@ -90,6 +102,21 @@ def polar_to_rectangular_dict(
         The inverse operation.
     local_gain_release_dict
         The analogous function for real-valued signals.
+
+    Examples
+    --------
+    .. plot::
+
+        >>> import plenoptic as po
+        >>> img = po.data.einstein()
+        >>> spyr = po.simul.SteerablePyramidFreq(img.shape[-2:])
+        >>> coeffs = spyr(img)
+        >>> energy, state = po.simul.non_linearities.rectangular_to_polar_dict(
+        ...     coeffs, residuals=True
+        ... )
+        >>> coeffs = po.simul.non_linearities.polar_to_rectangular_dict(energy, state)
+        >>> po.pyrshow(coeffs)
```

Comment:
could maybe show that these are identical to the original coeffs? but that would look fairly busy, what do you think?